setup.py

```diff
@@ -56,18 +60,25 @@ def get_requirements():
     )
 
 
-setup(
-    name='gcamwrapper',
-    version='0.1.0',
-    packages=find_packages(),
-    ext_modules=[gcam_module],
-    install_requires=get_requirements(),
-    url='https://github.com/JGCRI/gcamwrapper',
-    license='ECL 2',
-    author='Pralit Patel, Chris R. Vernon',
-    author_email='pralit.patel@pnnl.gov, chris.vernon@pnnl.gov',
-    description='Python API for GCAM',
-    long_description=readme(),
-    python_requires='>=3.6.*, <4'
-)
- 
+try:
+    # workaround to deal with R and Python being PITA about package data
+    shutil.copy(os.path.join('inst', 'extdata', 'query_library.yml'), 'gcamwrapper')
```

Comment:
I think we should include this as package data within the `gcamwrapper` package by creating a `data` dir and including the YAML file in there.  See previous comment. 

This is because all users will likely not want to clone the repository.  And if they are installing from GitHub using `pip` or from PyPI eventually, these files will not be included.

Reply:
We explored this a bit when installing on deception.  The approach here will work as intended when installing from GitHub or pip as the `query_library.yml` is getting copied in place before building the install package whichever it is wheel or egg.  Which I guess makes sense because I copied this approach from a suggestion someone else came up with who was trying to accomplish something similar.